src/main/java/ch/uzh/ifi/hase/soprafs22/controller/GameController.java

```diff
@@ -51,7 +51,7 @@ public List<CardGetDTO> getBlackCards(@RequestHeader(value = "authorization", re
     // In order to do that, first get current black cards, if there are any (in this case get empty list)
     cards = userService.getCurrentBlackCards(id);
     // if user has no current black cards, we get 8 new ones and assign them to the user for the next 24 hours
-    if (cards.isEmpty()){
+    if (cards == null || cards.isEmpty()){
```

Comment:
cards should never be null

Reply:
at least in the test it happend that it was null.. and there is no precondition visible that it will never be null.. if there is no cards == null there then there are different weird ways how the (test) program might fail.. thus I added it here as I understood the code that in case that it ever was null which it should never be then it should behave the same as if it was empty..